Metagenomics/tutorials/mothur-miseq-sop.md

```diff
@@ -0,0 +1,1516 @@
+>
+>    <details>
+>    <summary>Click to view answer</summary>
+>    23,488. <br>
+>    This can be determined by looking at the number of lines in bad.accnos output of screen.seqs
+>    or by comparing the total number of seqs between of the summary log before and after this screening
+>    step
+>    </details>
+{: .question}
+
+
+## Processing improved sequences
+
+### Optimize files for computation
+We anticipate that many of our sequences are duplicates of each other. Because it's computationally wasteful to align the same thing a bazillion times, we'll unique our sequences using the `unique.seqs` command:
+
+> ### :pencil2: Hands-on: Remove duplicate sequences
+>
+> - **Tool:** Unique.seqs
+> - **Parameters:**
+>   - Set **fasta** to the `good.fasta` output from Screen.seqs
+{: .hands_on}
+
+This tool outputs two files, one is a fasta file containing only the unique sequences, and a *names files*.
+The names file consists of two columns, the first contains the sequence names for each of the unique
+sequences, and the second column contains all other sequence names that are identical to the representative
+sequence in the first column.
+
+```
+name          representatives
+read_name1    read_name2,read_name,read_name5,read_name11
+read_name4    read_name6,read_name,read_name10
+read_name7    read_name8
+...
+```
+
+> ### :question: Question
+>
+> How many sequences were unique? how many duplicates were removed?
+>
+>    <details>
+>    <summary>Click to view answer</summary>
+>    16,426 unique sequences and 112,446 duplicates. <br>
+>    This can be determined from the number of lines in the fasta (or names) output, compared to the
+>    number of lines in the fasta file before this step. The log file also contains a line showing the
+>    total number of sequences before and the command: <br><br>
+>    mothur > unique.seqs(fasta=fasta.dat) <br>
+>    128872	16426
+>    </details>
+{: .question}
+
+To reduce file sizes further and streamline analysis, we can now summarize the data in a *count table*.
+
+> ### :pencil2: Hands-on: Generate count table
+>
+> - **Tool:** Count.seqs
+> - **Parameters:**
+>   - Set **name** to the `names` output from Unique.seqs
+>   - Set **Use a Group file** to `yes`
+>   - Set **group** to the group file we created using the Make.group tool
+{: .hands_on}
+
+The *count_table* output will look something like this:
+
+```
+Representative_Sequence                      total   F3D0   F3D1  F3D141  F3D142  ...
+M00967_43_000000000-A3JHG_1_1101_14069_1827  4402    370    29    257     142
+M00967_43_000000000-A3JHG_1_1101_18044_1900  28      1      0     1       0
+M00967_43_000000000-A3JHG_1_1101_13234_1983  10522   425    281   340     205
+...
+```
+
+The first column contains the read names of the representative sequence, and the subsequent columns contain
+the number of duplicates of this sequence observed in each sample.
+
+### Sequence Alignment
+We are now ready to align our sequences to the reference.
```

Comment:
it is what they recommend, I found this question asked on mothur forum as well iirc, will add their reasoning here :)